### PR TITLE
Updating GTEx Figure

### LINKED ISSUE #96: Update GTEx Figure to Include Correlation
Add correlation estimates for panels E and F

### PR DESCRIPTION
**Pull request by gwaygenomics**
closes #96

### CODE REVIEW

File: 8.gtex-interpret/scripts/nbconverted/4.visualize-signatures.r [hunk 1]
```diff
@@ -4,11 +4,14 @@ suppressPackageStartupMessages(library(dplyr))
 suppressPackageStartupMessages(library(ggplot2))
 suppressPackageStartupMessages(library(ggrepel))
 suppressPackageStartupMessages(library(cowplot))
+suppressPackageStartupMessages(library(ggpmisc))
 
 # Load custom plotting functions
 util_file = file.path("..", "6.analyze-weights", "scripts", "utils.R")
 source(util_file)
 
+set.seed(123)
```
[the remark this answers is not in the document]
Comment (gwaygenomics) [reply 8.gtex-interpret/scripts/nbconverted/4.visualize-signatures.r]: no, but I am adding a seed to control the `geom_jitter()` call

**Comment by gwaygenomics** [comment]
> For panel A: How are you selecting which data points are labeled? I assume this is based on a z-score cutoff

Yeah, selection based on a z-score cutoff.

> For panel F, its a little difficult to distinguish colors

Agree! I will update. Thanks for catching

> In the final combined figure you have some sub figure letters (C,E,F) that are overlapping axis labels, also some plots the text seems squashed (B,E), Not sure if the header is cutoff in G

Are you looking at the figure in the jupyter notebook or in the final `.pdf`? I don't see the squishing in the pdf

**Comment by gwaygenomics** [comment]
Thanks for comments @ajlee21 - can you check again in 9d1254e if this answers your concerns?

**Comment by ajlee21** [comment]
> Thanks for comments @ajlee21 - can you check again in [9d1254e](https://github.com/greenelab/BioBombe/commit/9d1254e058ca9f24e8ee096b01998a5e613c9d0c) if this answers your concerns?

I was looking at the figures in the NB, so in the pdf it looks ok.

**Pull request merged**